src/appointment/appointment.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { prop } from '@typegoose/typegoose';
+
+export class AppointmentDTO {
+    @ApiProperty()
+    @prop({ required: true })
+    startTime: Date;
+
+    @ApiProperty()
+    @prop({ required: true })
+    endTime: Date;
+
+    @ApiProperty()
+    @prop({ required: true })
+    location: string;
+
+    @ApiProperty()
+    @prop({ required: true })
+    price: number;
```

Comment:
`startTime` `endTime` `location` `price`  of `AppointmentDTO` can be extended from `EditAppointmentDTO` using `extends` to prevent code duplication.